src/components/navigation-bar/navigation-bar.spec.jsx

```diff
@@ -64,6 +64,12 @@ describe('Navigation Bar component', () => {
     expect(tabContent.props.title).toEqual('Custom Title 1');
   });
 
+  it('should have mountOnEnter prop set for every tab', () => {
+    const tabChildren = navigationBar.find(Tabs).props().children;
+    expect(tabChildren[0].props.mountOnEnter).toEqual(true);
```

Comment:
A forEach here might provide a good way to check this without having to know the length or anything specific about the tabs themselves.  
```javascript
const tabChildren = navigationBar.find(Tabs).props().children;

tabChildren.forEach((tab) => {
  expect(tab.props.mountOnEnter).toEqual(true);
});
```

Reply:
Hi @thescientist13 : Oh yes. This is cleaner. Will make the changes